scripts/install-frontend.sh

```diff
@@ -0,0 +1,4 @@
+cd src-frontend
+npm install -g bower
+npm install
+bower install
```

Comment:
nit, but can this go back to the previous directory when done? I feel like it won't matter all that much, but it is good to cleanup.